NEWS.md

```diff
@@ -26,6 +26,26 @@
      2.0  4.0
     ```
 
+* `NDArray` `getindex`/`setindex!` linear indexing support and `first` for extracting scalar value. (#TBD)
```

Comment:
`#TBD`?

Reply:
(at first I wrote this patch, the issue number was still unknown)
I will send a PR to sort them out for releasing.